test/arrayMax.test.js

```diff
@@ -0,0 +1,15 @@
+const { arrayMax } = require("../lib");
+
+describe("arrayMax", () => {
+  test("returns 24 as the max of the array", () => {
+    expect(arrayMax([5, 12, 3, 24])).toBe(24);
+  });
+
+  test("returns undefined with empty array", () => {
+    expect(arrayMax([])).toBe(undefined);
+  });
+
+  test("returns 0 with array of negatives and other types", () => {
```

Comment:
Hey @kyliedale - I'm just curious. The test description says to "return 0"; however, you are expecting a return of "undefined"?